Documentation/mgcb.md

```diff
@@ -50,13 +50,16 @@ Set the target platform for this build. It must be a member of the TargetPlatfor
 * Xbox360
 * iOS
 * Android
-* Linux
+* DesktopGL
 * MacOSX
 * WindowsStoreApp
 * NativeClient
+* PlayStationMobile
```

Comment:
You could just create a second PR, its a minor change so it will take you just a minute to set up...